src/Resources/Resources.Sdk/Readme.md

```diff
@@ -5,46 +5,138 @@ This directory contains management plane service clients of Az.Resources module.
-azure-arm: true
-license-header: MICROSOFT_MIT_NO_VERSION
+output-folder: Generated
+namespace: Microsoft.Azure.Management.Resources
+csharp:
+  clear-output-folder: false
+  reflect-api-versions: true
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
 ```
 
 
+## Configuration
 
-###
-``` yaml
+### Basic Information
+
+These are the global settings for the Resource API.
+
+``` yaml $(package-privatelinks)
+tag: package-privatelinks-2020-05
+```
+
+``` yaml $(package-features)
+tag: package-features-2021-07
+```
+
+``` yaml $(package-locks)
+tag: package-locks-2020-05
+```
+
+``` yaml $(package-resources)
+tag: package-resources-2021-04
+```
+
+``` yaml $(package-subscriptions)
+tag: package-subscriptions-2022-12
+```
+
+``` yaml $(package-managedapplications)
+tag: package-managedapplications-2018-06
+```
+
+``` yaml $(package-deploymentscripts)
+tag: package-deploymentscripts-2020-10
+```
+
+``` yaml $(package-templatespecs)
+tag: package-templatespecs-2021-05
+```
+
+``` yaml $(package-deploymentstacks)
+tag: package-deploymentstacks-2022-08-preview
+```
+
+### Tag: package-deploymentscripts-2023-08
+
+These settings apply only when `--tag=package-deploymentscripts-2020-10` is specified on the command line.
+
+```yaml $(tag) == 'package-deploymentscripts-2020-10'
 input-file:
-- Microsoft.Resources/preview/2022-08-01-preview/deploymentStacks.json
-- Microsoft.Resources/stable/2022-09-01/resources.json
-- Microsoft.Resources/stable/2021-01-01/subscriptions.json
-- Microsoft.Resources/stable/2020-10-01/deploymentScripts.json
-- Microsoft.Resources/stable/2021-05-01/templateSpecs.json
-- Microsoft.Resources/stable/2016-09-01/links.json
-- Microsoft.Authorization/stable/2020-09-01/dataPolicyManifests.json
-- Microsoft.Authorization/stable/2020-09-01/policyAssignments.json
-- Microsoft.Authorization/stable/2020-09-01/policyDefinitions.json
-- Microsoft.Authorization/stable/2020-09-01/policySetDefinitions.json
-- Microsoft.Authorization/preview/2020-07-01-preview/policyExemptions.json
-- Microsoft.Authorization/stable/2020-05-01/locks.json
-- Microsoft.Authorization/stable/2020-05-01/privateLinks.json
-- Microsoft.Features/stable/2021-07-01/features.json
-- Microsoft.Features/stable/2021-07-01/SubscriptionFeatureRegistration.json
+- https://github.com/Azure/azure-rest-api-specs/blob/main/specification/resources/resource-manager/Microsoft.Resources/stable/2020-10-01/deploymentScripts.json
 
+suppressions:
+  - code: OperationsAPIImplementation
+    reason: OperationsAPI will come from Resources
+```
 
+### Tag: package-resources-2021-04
 
-output-folder: Generated
+These settings apply only when `--tag=package-resources-2021-04` is specified on the command line.
 
-namespace: Microsoft.Azure.Management.Resources
+``` yaml $(tag) == 'package-resources-2021-04'
+input-file:
+  - https://github.com/Azure/azure-rest-api-specs/tree/main/specification/resources/resource-manager/Microsoft.Resources/stable/2021-04-01/resources.json
```

Comment:
Instead of using main branch, please use commit id to pin the swagger. And there are similar issue in following code.

Reply:
Made changes to the PR according to your suggestions. Please let us know if there is anything else we can do.